src/analysis/retail/shaman/enhancement/modules/talents/ElementalAssault.tsx

```diff
@@ -65,19 +71,21 @@ class ElementalAssault extends Analyzer {
         position={STATISTIC_ORDER.OPTIONAL()}
         category={STATISTIC_CATEGORY.TALENTS}
         size="flexible"
+        tooltip={
+          <>
+            {this.maelstromWeaponGained} Total Maelstrom Gained
+            <br />
+            {this.maelstromWeaponWasted} Maelstrom Wasted
+            <hr />
+            {this.maelstromWeaponGained - this.maelstromWeaponWasted} Effective Maelstrom Gained
```

Comment:
ideally, use `<p>` or `<div/>` rather than explicit line breaks. `<hr />` has a visual element as well so it is fine.

```suggestion
            <div>{this.maelstromWeaponGained} Total Maelstrom Gained</div>
            <div>{this.maelstromWeaponWasted} Maelstrom Wasted</div>
            <hr />
            <div>{this.maelstromWeaponGained - this.maelstromWeaponWasted} Effective Maelstrom Gained</div>
```